src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 2 failures in `PowerShell-CI-macos`
Experimental Feature: && and || operators - Feature-Enabled.Handles nested assignment
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 167
167:         $x[0] | Should -BeExactly '0'

```
Experimental Feature: && and || operators - Feature-Enabled.Evaluates assignment correctly in conditions
```powershell
Argument type cannot be System.Void.
Parameter name: type
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/It.ps1: line 281
281:                     $null = & $ScriptBlock @Parameters

```